internal/source/unpacker.go

```diff
@@ -112,5 +123,10 @@ func NewDefaultUnpacker(mgr ctrl.Manager, namespace, provisionerName, unpackImag
 			Client: mgr.GetClient(),
 			reader: mgr.GetAPIReader(),
 		},
+		rukpakv1alpha1.SourceTypeUpload: &Upload{
+			baseDownloadURL: baseUploadManagerURL,
+			bearerToken:     mgr.GetConfig().BearerToken,
+			client:          http.Client{Timeout: 10 * time.Second, Transport: httpTransport},
```

Comment:
Any value in a constant variable for the timeout period?

Reply:
Sure, why not!